specification/src/main/asciidoc/webprofile/RevisionHistory.adoc

```diff
@@ -1,61 +1,61 @@
+[appendix]
 == Revision History
```

Comment:
Same comment here as for the platform spec.

Reply:
Yep, and same answer...  :-)  I guess we need a wider audience to help resolve this question.

Reply:
Please bring this up on one of the mailing lists.